digest/glossary.js

```diff
@@ -0,0 +1,28 @@
+const utils = require('./utils')
+
+module.exports = () =>
+  utils.readdir('/glossary')
+    .then(convertGlossaryFileNamesToTerms)
+    .then(indexById)
+
+const convertGlossaryFileNamesToTerms = glossaryFileName =>
+  glossaryFileName
+    .map(fileName => ({
+      fileName,
+      id: fileName,
+      value: fileName.replace(/-/g, ' '),
+      path: `/glossary/${fileName}`,
+      markdownFilePath: `/glossary/${fileName}/README.md`,
```

Comment:
were not using this property. lets remove it

Reply:
We're using the 'path' property in our views. 

'markdownFilePath' is needed per spec. Should we remove it?